### DISCUSSION TITLE
ArgumentCountError

### DISCUSSION THREAD

**Discussion by DaveMolla** [filamentphp/filament#3942] [Help] [answered]
too few arguments to function filament\widgets\statsoverviewwidget\card::make()

I'm getting this error when I try to use card
![Screenshot (278)](https://user-images.githubusercontent.com/92756138/189492386-d209d6a8-4e63-403f-b337-e649b3dbebbe.png)

**Comment by danharrin** [comment, marked as answer]
You've imported the wrong Card class, it should be from the forms package, not widgets

**Reply by DaveMolla** [reply]
Thank You my Friend!
